src/Core/src/Properties/AssemblyInfo.cs

```diff
@@ -10,6 +10,9 @@
 [assembly: InternalsVisibleTo("Microsoft.Maui.Controls.Compatibility.Tizen")]
 [assembly: InternalsVisibleTo("Microsoft.Maui.Controls.Compatibility.ControlGallery")]
 [assembly: InternalsVisibleTo("Microsoft.Maui.Controls.Compatibility.ControlGallery.Android")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Controls.Android.UITests")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Controls.iOS.UITests")]
+[assembly: InternalsVisibleTo("WinUI.UITests")]
```

Comment:
Not sure we want this. If you need to override the dispatcher, then you should use the keyed service `IDispatcher` with key `typeof(IApplication)`.

Once we start making core expose internals to more things, then we get even more spaghetti - and this file makes me cry all night. Keyed services are the correct way.

But, we we need this in ASAP... then we might as well add some more code debt, we got lots of it already and we need to do some cleanup in net102322.0

Reply:
Yea, this is largely just a hack while we're stuck with our legacy tests. 

Because it's a shared project, when the tests instantiate the test class they also instantiate XPLAT elements like `TabbedPage` which access a dispatcher they don't even use. 

So, ideally this is a "hack" that will solve itself once we get over to Appium completely :-)